src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.validation.annotation.Email;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "^(?=.{1,64}@)[A-Za-z0-9_-]"
+            + "+(\\.[A-Za-z0-9_-]+)*@[^-][A-Za-z0-9-]"
+            + "+(\\.[A-Za-z0-9-]+)*(\\.[A-Za-z]{2,})$";
```

Comment:
Let's create a constant of a type Pattern but not a String. Because otherwise there will be created a new object of this type each time you call the method `matches`